packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-pwd-recovery-mfa/java/integrationsteps.md

```diff
@@ -1,3 +1,98 @@
 ## Integration steps
 
-The Java SDK integration steps for this use case are coming soon. See the [Okta Identity Engine Java SDK usage guide](https://github.com/okta/okta-idx-java#usage-guide) for more information.
+### Step 1: User clicks forgot password link
+
+The password recovery flow begins when the user clicks the **Forgot your password?** link on your app's sign-in page. Create a **Forgot your password?** link that directs the user to a reset my password form, such as the following example:
+
+<div class="common-image-format">
+
+![Displays the Sign-in form for Java SDK with a 'Forgot your password?' link](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-forgot-java.png)
+
+</div>
+
+You need to create a form to capture the user's email for password recovery, such as the following reset my password form:
+
+<div class="common-image-format">
+
+![Displays the reset password form for Java SDK with an email field](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-reset-java.png)
+
+</div>
+
+Begin the authentication process by calling Java SDK's [`IDXAuthenticationWrapper.begin()`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java#L603) method and getting a new [`ProceedContext`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/ProceedContext.java) object.
+
+```java
+AuthenticationResponse beginResponse = idxAuthenticationWrapper.begin();
+ProceedContext proceedContext = beginResponse.getProceedContext();
+```
+
+### Step 2: User enters their email
+
+After the user submits their email, call the [`IDXAuthenticationWrapper.recoverPassword()`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java#L177) method, passing in the user's email.
+
+```java
+ AuthenticationResponse authenticationResponse =
+    idxAuthenticationWrapper.recoverPassword(username, proceedContext);
+```
+
+If the email is for a valid and active user in the Okta directory, the `IDXAuthenticationWrapper.recoverPassword()` method returns an [`AuthenticationResponse`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/response/AuthenticationResponse.java) object with the following properties:
+
+1. `AuthenticationStatus` = `AWAITING_AUTHENTICATOR_SELECTION` <br>
```

Comment:
I will change it to `AuthenticationStatus=AWAITING_AUTHENTICATOR_SELECTION`. This is a state of the status, not a definition of AuthenticationStatus. Although I it would be good to have a standard for code term followed by definition. I sometimes see em-dash and colon used in the guides, and API references.

Reply:
Ah, I understand now; thanks for the clarification! Your updated version looks fine, and this comment can be resolved.